rspirv/sr/module.rs

```diff
@@ -61,8 +61,11 @@ pub struct Module {
     /// All constants.
     pub constants: Storage<Constant>,
 
-    /// All functions.
-    pub functions: Vec<Function>,
+    /// All basic blocks.
+    pub basic_blocks: Storage<BasicBlock>,
```

Comment:
What is the reasoning behind having shared block storage between functions rather than per function? Having it per function would allow each function to be treated more separately, and moving blocks between pre-existing functions isn't exactly going to be a common operation.

Reply:
I think you are correct. These should be moved to a function.